Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -99,7 +99,15 @@ - (nullable UIActivityViewController *)sharingActivityViewControllerWithTextSnip
         [items addObject:[[NSURL alloc] initWithString:[NSString stringWithFormat:@"%@?%@", url.absoluteString, @"wprov=sfsi1"]]];
     }
 
-    UIActivityViewController *vc = [[UIActivityViewController alloc] initWithActivityItems:items applicationActivities:@[[[TUSafariActivity alloc] init]]];
+    if ([self mapItem]) {
```

Comment:
This might be overkill, but I usually do something along the lines of 
```
MKMapItem *mapItem = [self mapItem];
if (mapItem) {
    [items addObject:mapItem];
}
```
In case there's every anything funky about the getter that returns nil even after it just didn't return nil